Misc/NEWS.d/next/Library/2024-10-13-15-04-58.gh-issue-125398.UW7Ndv.rst

```diff
@@ -0,0 +1 @@
+Fix the conversion of the VIRTUAL_ENV path in the activate script in :mod:`venv` when running in Git Bash for Windows.
```

Comment:
Recommend used this.
```suggestion
Fix the conversion of the ``VIRTUAL_ENV`` path in the activate script in :mod:`venv` when running in Git Bash for Windows.
```

Reply:
Actually, the correct markup would be `:envvar:` :) See also the relevant parts of [the devguide](https://devguide.python.org/documentation/markup/).

Reply:
I edited the PR to use the `:envvar:` role, but it seems to fail documentation linting:
```
WARNING: 'envvar' reference target not found: VIRTUAL_ENV [ref.envvar]
```
Perhaps a nits file needs updating? @erlend-aasland, any ideas?

Reply:
You can use ``` :envvar:`!VIRTUAL_ENV` ``` :)

Reply:
Ideally though, `VIRTUAL_ENV` should be documented in `Doc/library/venv.rst`.

Reply:
Pretty sure `VIRTUAL_ENV` is undocumented on purpose. It's `sys.argv[0]` that determines whether you're in a venv or not (and if `argv[0]` is a relative path, it's resolved using `PATH`, which makes that the other relevant environment variable).

Reply:
Ah, but it's not _undocumented_; it is mentioned multiple places in our docs, _but_ it lacks a "formal" `.. envvar::` definition ;)

Reply:
Looking at the documentation in the venv module for it, I'd be okay with adding the reference location to there (in short, it says "don't use or rely on this variable, but it might be set")

Reply:
OK, I'm minded to commit this as it is currently (with `!VIRTUAL_ENV`), and that `envvar` definition can be added in a subsequent PR. Any objections?